src/js/state/rootReducer.ts

```diff
@@ -27,7 +27,7 @@ export default combineReducers<any, any>({
   tabs: Tabs.reducer,
   investigation: Investigation.reducer,
   view: View.reducer,
-  spaces: Spaces.reducer,
+  pools: Pools.reducer,
```

Comment:
we will also need to write a migration here to change existing states with `spaces` -> `pools`

Reply:
You actually don't want to do this, the old spaces should be migrated via the brimcap migrate command (in progress).

Reply:
Good catch. Folks can read up on the current plan at #1567 and brimdata/brimcap#58.